spec/lib/update_school_data_spec.rb

```diff
@@ -119,6 +132,19 @@
         expect(school.northing).to eql('181201')
         expect(school.geolocation.x).to be_within(0.0000000000001).of(51.51396894535262)
         expect(school.geolocation.y).to be_within(0.0000000000001).of(-0.07751626505544208)
+        expect(school.status).to eql('Open')
```

Comment:
Could these expectancies be refactored into a method?

Reply:
what kind of method did you have in mind?

Reply:
Not sure. Sorry for replying late @dushan-madetech 
These lines of expectations are in two different tests, I think. 
So I was wondering if there was a way to extract all the expectancies into a method and then call the method in the tests so that we could reduce the number of lines. 🤔